crates/bevy_ecs/src/reflect/entity_commands.rs

```diff
@@ -181,6 +181,27 @@ impl<'w, 's, 'a> ReflectCommandExt for EntityCommands<'w, 's, 'a> {
     }
 }
 
+/// Helper function to add a reflect component to a given entity
+fn insert_reflect(
+    world: &mut World,
+    entity: Entity,
+    type_registry: &TypeRegistry,
+    component: Box<dyn Reflect>,
+) {
+    let type_info = component.type_name();
+    let Some(mut entity) = world.get_entity_mut(entity) else {
+        panic!("error[B0003]: Could not insert a reflected component (of type {}) for entity {:?} because it doesn't exist in this World.", component.type_name(), entity);
```

Comment:
```suggestion
        panic!("error[B0003]: Could not insert a reflected component (of type {}) for entity {entity:?} because it doesn't exist in this World.", component.type_name());
```